platform/rct/hal/usb_stm32/hw_config.c

```diff
@@ -213,6 +214,7 @@ void USB_Interrupts_Config(void)
   NVIC_Init(&NVIC_InitStructure);
 
 #elif defined(STM32F37X)
+#error "Why are we here?"
   /* Enable the USB interrupt */
```

Comment:
If we're going to just throw an error here, we need to gut these.  The linenoise doesn't add anything if we're not going to port to these platforms in the future.